js/application.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
+	
+	var headerTemplate = $.get('js/templates/header.handlebars', function(data){
```

Comment:
This copy-paste is probably a good sign to make a reusable function. The only difference is the file and id of the div, so that could be parameterized and called across an array.